btr-web/btr-main-app/cypress/e2e/accessibility/ownerChange.cy.ts

```diff
@@ -95,21 +95,44 @@ describe('accessibility -> Beneficial Owner Change', () => {
     // cy.checkA11y('[data-cy=isUnableToObtainOrConfirmInformation]')
 
     // summary table 19444
-    // cy.checkA11y('[data-cy=individualsSummaryTable]')
-    // open action buttons
-    // cy.get('[data-cy=popover-button]').eq(0).click()
-    // cy.checkA11y('[data-cy=individualsSummaryTable]')
+    cy.checkA11y('[data-cy=individualsSummaryTable]', { rules: { 'nested-interactive': { enabled: false } } })
```

Comment:
better to move this into its own `it()` test I think

Reply:
Sure. I'll update the test